### PR TITLE
Update index.html.md

### PR DESCRIPTION
**Pull request by bblinder**
## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

File: content/source/docs/cloud/agents/index.html.md
Comment (vaficionado) [inline content/source/docs/cloud/agents/index.html.md:192]:
```suggestion
You may choose to run multiple agents within your network, up to the organization's purchased agent limit. If there are multiple agents available within an organization, Terraform Cloud will select the first available agent within the target pool.
```

File: content/source/docs/cloud/agents/index.html.md
Comment (vaficionado) [inline content/source/docs/cloud/agents/index.html.md:196]:
```suggestion
If an agent terminates abnormally, any running jobs can be restarted where they left off by resubmitting them. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```

File: content/source/docs/cloud/agents/index.html.md
Comment (vaficionado) [inline content/source/docs/cloud/agents/index.html.md:216]:
```suggestion
Agents are considered active and count towards the organization's purchased agent capacity if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organizations total agent capacity.
```

**Review by vaficionado — CHANGES_REQUESTED** [review]
Tweaked some of the wording; if the changes proposed are accepted I can approve this from my end. Thanks for taking the time to put this together. Also adding @nfagerlund as reviewer for final say.

**Comment by bblinder** [comment]
@vaficionado  -- this changes work for me!

**Comment by vaficionado** [comment]
Cool, would you mind merging them to the PR so Nick can see the updated version when they review?

**Comment by bblinder** [comment]
Done.

**Review by vaficionado — APPROVED** [review]
LGTM, thanks for the changes!

File: content/source/docs/cloud/agents/index.html.md
Comment (nfagerlund) [inline content/source/docs/cloud/agents/index.html.md:216]:
```suggestion
Agents are considered active and count towards the organization's purchased agent capacity if they are in the **Idle**, **Busy**, or **Unknown** state. Agents which are in the **Errored** or **Exited** state do not count towards the organization's total agent capacity.
```

**Review by nfagerlund — APPROVED** [review]
Spotted one missing apostrophe, otherwise lgtm! I'll go ahead and apply that patch myself and merge this.

**Pull request merged**
